ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -96,6 +96,40 @@ def get_table_item(
             raise ApiError(response.text, response.status_code)
         return response.json()
 
+    def aggregator_value(
```

Comment:
wow lol what a function, seems like something the API should support maybe. good for now though.

Reply:
I think it would probably make sense to make a Token API as well.